lib/cartopy/examples/reprojected_wmts.py

```diff
@@ -35,7 +35,7 @@ def plot_city_lights():
 
     # Plot WMTS data in a specific region, over a plain lat-lon map.
     ax = fig.add_subplot(1, 2, 1, projection=plain_crs)
-    ax.set_extent((-6, 3, 48, 58), crs=ccrs.PlateCarree())
+    ax.set_extent((-6, 3, 48, 58), crs=plain_crs)
```

Comment:
This one doesn't need changing. For me this change reinforces the false perception that the projection and crs should be the same.